src/pages/Login/Login.js

```diff
@@ -1,23 +1,71 @@
-import React from 'react';
+import React, { useState } from 'react';
+import { Navigate, useNavigate } from 'react-router-dom';
 import { Link } from 'react-router-dom';
 import './Login.scss';
 
 const Login = () => {
+  const [id, setId] = useState('');
+  const [pw, setPw] = useState('');
+
+  const navigate = useNavigate();
+
+  const goMainOk = id.includes('@') && pw.length >= 8;
+  const goMainBtn = goMainOk ? false : true;
+
   return (
     <div className="login">
       <div className="login-box">
         <h1 className="login-logo">로그인</h1>
         <form className="login-form">
           <span className="email-login">이메일 로그인</span>
           <input
+            onChange={e => {
+              setId(e.target.value);
+            }}
             className="id-box"
             placeholder="아이디(이메일 주소)를 입력하세요"
+            value={id}
           />
-          <input className="password-box" placeholder="비밀번호를 입력하세요" />
-          <button className="login-button">로그인</button>
+          <input
+            onChange={e => {
+              setPw(e.target.value);
+            }}
+            className="password-box"
+            type="password"
+            placeholder="비밀번호를 입력하세요"
+            value={pw}
+          />
+          <button
+            onClick={e => {
+              e.preventDefault();
+              fetch('http://10.58.52.125:8000/users/signin', {
+                method: 'POST',
+                headers: { 'content-Type': 'application/json;charset=utf-8' },
+                body: JSON.stringify({
+                  email: id,
+                  password: pw,
+                }),
+              })
+                .then(res => res.json())
+                .then(data => {
+                  if (data.access_token) {
+                    localStorage.setItem('token', data.access_token);
+
+                    alert('환영합니다!');
+                    navigate('/main');
+                  } else if (data.message === 'specified user does not exist') {
+                    alert('존재하지 않는 유저입니다.');
+                  }
+                });
+            }}
+            className={goMainOk ? 'login-button' : 'login-button-no'}
+            disabled={goMainBtn}
```

Comment:
```suggestion
            disabled={!goMainOk}
```
not연산자 사용하시면 되는데 일단, goMainOk 변수명이 어썸하지 않아서 변경부터 해주세요.